closure/testing/phantomjs_harness.js

```diff
@@ -189,6 +189,14 @@ function onLoadFinished(status) {
   }
 }
 
+/**
+ * Callback when a script fails to load in time.
+ * @param {!phantomjs.Server.ResourceError} resource
+ */
+function onResourceTimeout(resource) {
+    system.stderr.writeLine('Resource timed out: ' + resource.url);
+    retry();
```

Comment:
We probably should use Bazels flaky/retry mechanism rather than silently retrying tests, but I see that we're doing it already in some case(s), so I'm fine with doing it here as well.